tests/language-feature/enums/unscoped-enum-explicit-type copy.slang

```diff
@@ -0,0 +1,27 @@
+//TEST(compute):COMPARE_COMPUTE(filecheck-buffer=CHECK): -shaderobj
```

Comment:
Can you rename the "file name" properly?

Reply:
I accidently added the file, there was only supposed to be 1